### PR TITLE
Add Palindrome Number in Kotlin

### PR DESCRIPTION
**Pull request by hemant2705**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes
(Write your answer here.)

### Description
Checking whether the number is Palindrome or not in kotlin.
(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#4949 
(Write your answer here.)

### CODE REVIEW

**Comment by hemant2705** [comment]
Can Anyone please review it?

File: Kotlin/Maths/PalindromeNumber.kt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+Run 2:
+Enter Number  :
+343
+343 is a Palindrome Number
+---
+Run 3:
+Enter Number  :
+2344
+2344 is not a Palindrome Number
+*/
```
Comment (HarshCasper) [inline Kotlin/Maths/PalindromeNumber.kt]: Have only one test case. Add a new line at the EOF

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by hemant2705** [comment]
I committed changes as requested .
Can you go through it please.

File: Kotlin/Maths/PalindromeNumber.kt (hunk 1 above)
Comment (hemant2705) [reply Kotlin/Maths/PalindromeNumber.kt]: Applied changes

**Review by ankitaggarwal23 — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**